pkg/object/objects.go

```diff
@@ -410,6 +411,123 @@ func ManifestDiff(a, b string) (string, error) {
 	return sb.String(), err
 }
 
+// buildResourceRegexp translates the resource indicator to regexp.
+func buildResourceRegexp(s string) *regexp.Regexp {
+	hash := strings.Split(s, ":")
+	for i, v := range hash {
+		if v == "" {
+			hash[i] = ".*"
+		}
+	}
+	return regexp.MustCompile(strings.Join(hash, ":"))
```

Comment:
This will panic if it fails so we can't use it here.

Reply:
Yes, changed to `regexp.Compile(strings.Join(hash, ":"))`